packages/react-core/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -80,7 +80,8 @@ export const ToolbarItem: React.FunctionComponent<ToolbarItemProps> = ({
         <div
           className={css(
             styles.toolbarItem,
-            variant && styles.modifiers[toCamel(variant) as 'pagination' | 'label' | 'chipGroup'],
+            variant && styles.modifiers[toCamel(variant) as 'pagination' | 'label'],
+            variant === ToolbarItemVariant['chip-group'] && styles.modifiers.labelGroup,
```

Comment:
Could we instead update the `chip-group` variant to `label-group` for the ToolbarItemVariant enum and prop type?

Reply:
Opened https://github.com/patternfly/patternfly-react/issues/10405 to track that change in a follow up, along with the rest of Toolbar (as there are several usages of "chip" in its language for prop/component names).